src/main/scala/com/github/unchama/seichiassist/subsystems/gacha/bukkit/GachaCommand.scala

```diff
@@ -195,14 +196,23 @@ class GachaCommand[
 
     val add: ContextualExecutor =
       playerCommandBuilder
-        .argumentsParsers(List(probabilityParser))
+        .argumentsParsers(List(probabilityParser, Parsers.identity))
         .execution { context =>
           val player = context.sender
-          val probability = context.args.parsed.head.asInstanceOf[Double]
+          val args = context.args.parsed
+          val probability = args.head.asInstanceOf[Double]
+          val eventName =
+            Option.when(args(1).toString != null)(GachaEventName(args(1).toString))
```

Comment:
```suggestion
            Option(args(1).toString).map(GachaEventName)
```
https://www.scala-lang.org/api/2.13.3/scala/Option$.html#apply[A](x:A):Option[A] は`null`なら`None`を返します